src/constants.js

```diff
@@ -32,7 +32,9 @@ export const DTYPE_VALUES = {
     format: GL.R32F,
     dataFormat: GL.RED,
     type: GL.FLOAT,
-    max: 2 ** 31 - 1,
+    // Not sure what to do about this one - a good use case for channel stats, I suppose:
+    // https://en.wikipedia.org/wiki/Single-precision_floating-point_format.
+    max: 3.4 * 10 ** 38,
```

Comment:
Pretty confused here. So single precision float is different from float32? We end up using a `Float32Array` to take views of the underlying buffer, so how can it be different? Sorry for my nativity.

Reply:
Also, don't `DTYPE_VALUES` get reused for zarr? If so, zarr's `<f4` is `max: 2 ** 31 - 1` and not single float-point format max?

Reply:
Could you link to the `zarr` reference for that?  I think this is pretty standard and that sounds like the maximum for a 32 bit signed integer.

I don't think there is such thing as the `float32` name in OMEXML - single precision is a 32 bits of precision float and is called "float" and a "double" is 64 bits of precision for a floating point number: https://en.wikipedia.org/wiki/Double-precision_floating-point_format (and then "half", which is not in the OMEXML specification, is 16: https://en.wikipedia.org/wiki/Half-precision_floating-point_format).  Does this answer?  Maybe I am not understanding your confusion, and maybe I am confused (I often am).

Reply:
I can clarify the comment - it was more that I wasn't sure what to do about such a large number since any slider would not be very usable with that sort of range.